cmd/hatchet-migrate/migrate/migrations/20250722162812_v1_0_30.sql

```diff
@@ -0,0 +1,24 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TYPE v1_payload_type AS ENUM ('WORKFLOW_INPUT', 'TASK_OUTPUT');
+
+CREATE TABLE v1_payload (
+    tenant_id UUID NOT NULL,
+    task_id BIGINT NOT NULL,
+    task_inserted_at TIMESTAMPTZ NOT NULL,
```

Comment:
unsure if this is a good idea or not - I was trying to think of cases where we'd want to tie payloads to non-task things but was struggling to think of any